test/_unit/server.spec.js

```diff
@@ -44,6 +45,7 @@ describe('Server.js app file', () => {
       './apps/cepr': appsCeprStub,
       './apps/verify': appsVerifyStub,
       'hof/components/clear-session': behavioursClearSessionStub,
+      './apps/ima/behaviours/set0': behavioursSetStaticEmailsStub,
```

Comment:
is this the correct file path?

Reply:
Accidentally left a mistake in the name. Changed and re-ran the tests.